packages/Popover/stories/examples/Variations.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { CenteredStory, Gap, Tagline, Rule, Story } from "storybook/assets/styles/common.styles";
+import Button from "@paprika/button";
+import Popover from "@paprika/popover";
+import ButtonExample from "./ButtonExample";
+import ButtonIsEagerExample from "./ButtonIsEagerExample";
+import ShouldKeepFocusExample from "./ShouldKeepFocusExample";
+
+const Variations = () => (
+  <CenteredStory>
+    <div>
+      <Tagline>Popover Button</Tagline>
+      <Story>
+        <ButtonExample />
+      </Story>
+      <Gap />
+      <Rule />
+      <Tagline>Popover isEager</Tagline>
+      <Story>
+        <ButtonIsEagerExample />
+      </Story>
+      <Gap />
+      <Rule />
+      <Tagline>PopOver isDark</Tagline>
+      <Story>
+        <Popover isEager isDark defaultIsOpen>
+          <Popover.Trigger data-test-attr="propagated">
+            <span role="img" aria-label="bulb light">
+              💡
+            </span>
+          </Popover.Trigger>
+          <Popover.Content>
+            <Popover.Card>Lorem ipsum single-origin kombucha.</Popover.Card>
+          </Popover.Content>
+        </Popover>
+      </Story>
+      <Gap />
+      <Rule />
+      <Tagline>Popover shouldKeepFocus</Tagline>
+      <Story>
+        <ShouldKeepFocusExample />
+      </Story>
+      <Gap />
+      <Rule />
+      <Tagline>Popover defaultIsOpen</Tagline>
```

Comment:
Let's save this example for its own example story, and try using `defaultIsOpen` on all of the examples here on Variations so that when the page opens you can see everything at a glance.

Reply:
... but still get a feel for the interactions.